app/v2/controllers/ConvertError.scala

```diff
@@ -67,6 +67,7 @@ trait ConvertError extends Logging {
       case UnexpectedError(thr)                              => PresentationError.internalServiceError(cause = thr)
       case InvalidMessageTypeError(messageType)              => PresentationError.badRequestError(s"$messageType is not a valid message type")
       case ParsingError(message)                             => PresentationError.badRequestError(message)
+      case BusinessValidationError(message)                  => PresentationError.badRequestError(message)
```

Comment:
This is wrong. The code sent to the trader should be `BUSINESS_VALIDATION_ERROR`, as per the ticket, not `BAD_REQUEST`